python/py-fido2/Portfile

```diff
@@ -21,13 +21,14 @@ checksums           rmd160  0c8ba2335962501c9da55b24d1bbe2f4a1c63e87 \
                     sha256  2b4b4e620c2100442c20678e0e951ad6d1efb3ba5ca8ebb720c4c8d543293674 \
                     size    243526
 
-python.versions     37 38 39 310
+python.versions     37 38 39 310 311
 
 if {${name} ne ${subport}} {
     depends_build-append \
         port:py${python.version}-setuptools
 
     depends_lib-append \
         port:py${python.version}-six \
-        port:py${python.version}-cryptography
+        port:py${python.version}-cryptography \
+        port:poetry
```

Comment:
Why is poetry needed? Is it only needed on Python 3.11?

Reply:
I haven't looked at the port, but the following is probably required instead:

```tcl
python.pep517         yes
python.pep517_backend poetry
```

pep517 builds have been set as the default for python 3.11.

https://github.com/macports/macports-ports/blob/4be7972618122de4fbef67cc973875bccab5446e/_resources/port1.0/group/python-1.0.tcl#L351

Reply:
if that's the case ideally you would update the port to always use "python.pep517" also for the older subports.

Reply:
Updated in latest revision. I had to add py-idna to the list of dependencies as `port install -vst` would fail otherwise